### PR TITLE
Add a GUI to showcase the abilities of the HVAC service

### PR DESCRIPTION
**Pull request by lukasmittag**
GUI is implemented through the DearPyGUI (licenses need to be added before merging). It mocks a sensor and an application in the car or on a phone. Very lightweight and nothing too special.

### CODE REVIEW

**Comment by SebastianSchildt** [comment]
Like the idea. Things I noticed: Should probably add all needed dependencies to Pipfile, missing was

```
grpcio
dearpygui
protobuf
```

dearpygui itself should not be a problem as new dependency (MIT),


When looking into the API migration we might also want to think about, whether we want to port/rewrite this demo in "Pure VSS" style, as this should be quite easy in this example. 

Found a little crash when putting the "Ratio to poll sensor" to 0

```
    if (count % sensor_ratio) == 0:
ZeroDivisionError: integer division or modulo by zero
``

File: hvac_service/hvac_gui.py [hunk 1]
```diff
@@ -0,0 +1,76 @@
+import dearpygui.dearpygui as dpg
+import dearpygui.demo as demo
+import testclient
+
+def main():
+    client = testclient.HVACTestClient("127.0.0.1:50052")
```
Comment (romainletendart) [inline hvac_service/hvac_gui.py:6]:
```suggestion
    hvac_addr = os.getenv("HVAC_ADDR", "127.0.0.1:50052")
    client = testclient.HVACTestClient(hvac_addr )
```
Let's make this configurable like in testclient.py.

File: hvac_service/README.md [hunk 2]
```diff
@@ -29,3 +29,9 @@ The HVAC service is a service dummy allowing to control the state of the A/C and
 Configuration options have the following priority (highest at top):
 1. environment variable
 1. default value
+
+For using the HVAC GUI you need to have dearpygui installed: 
+
+''' pip install dearpygui '''
```
Comment (romainletendart) [inline hvac_service/README.md:35]: I would move this to requirements.txt.

File: hvac_service/hvac_gui.py [hunk 3]
```diff
@@ -0,0 +1,78 @@
+import dearpygui.dearpygui as dpg
```
Comment (SebastianSchildt) [inline hvac_service/hvac_gui.py:1]: This needs a license header

**Review by SebastianSchildt — CHANGES_REQUESTED** [review]
See comments

**Comment by lukasmittag** [comment]
Since there is the android companion app closing this

**Pull request closed without merging**